app/models/designation.js

```diff
@@ -61,6 +62,58 @@ export default Model.extend({
     return this.get("state").capitalize();
   }),
 
+  stateIcon: Ember.computed('stateIcon', function () {
```

Comment:
@patrixr , I think this computed property should be dependent on `order.state` ?
What do you think?